qiskit_experiments/calibration/calibrations.py

```diff
@@ -0,0 +1,698 @@
+        schedule: Schedule = None,
+    ) -> QuantumCircuit:
+        """
+        Queries a schedule by name for the given set of qubits. The parameters given
+        under the list free_params are left unassigned. The queried schedule is then
+        embedded in a gate with a calibration and returned as a quantum circuit.
+
+        Args:
+            schedule_name: The name of the schedule to retrieve.
+            qubits: The qubits for which to generate the gate with the schedule in it.
+            free_params: Names of the parameters that will remain unassigned.
+            group: The calibration group from which to retrieve the calibrated values.
+                If unspecified this defaults to 'default'.
+            schedule: The schedule to add to the gate if the internally stored one is
+                not used.
+
+        Returns:
+            A quantum circuit in which the parameter values have been assigned aside from
+            those explicitly specified in free_params.
+        """
+        if schedule is None:
+            schedule = self.get_schedule(schedule_name, qubits, free_params, group)
+
+        gate = Gate(name=schedule_name, num_qubits=len(qubits), params=list(schedule.parameters))
+        circ = QuantumCircuit(len(qubits), len(qubits))
+        circ.append(gate, list(range(len(qubits))))
+        circ.add_calibration(gate, qubits, schedule, params=schedule.parameters)
+
+        return circ
+
+    def schedules(self) -> List[Dict[str, Any]]:
+        """
+        Return the schedules in self in a list of dictionaries to help
+        users manage their schedules.
+
+        Returns:
+            data: A list of dictionaries with all the schedules in it.
+        """
+        data = []
+        for context, sched in self._schedules.items():
+            data.append({"context": context, "schedule": sched, "parameters": sched.parameters})
+
+        return data
+
+    def parameters_table(
+        self,
+        parameters: List[str] = None,
+        schedules: List[Union[Schedule, str]] = None,
+        qubit_list: List[Tuple[int, ...]] = None,
+    ) -> List[Dict[str, Any]]:
+        """
+        A convenience function to help users visualize the values of their parameter.
+
+        Args:
+            parameters: The parameter names that should be included in the returned
+                table. If None is given then all names are included.
+            schedules: The schedules to which to restrict the output.
+            qubit_list: The qubits that should be included in the returned table.
+                If None is given then all channels are returned.
+
+        Returns:
+            data: A dictionary of parameter values which can easily be converted to a
+                data frame.
+        """
+
+        data = []
+
+        # Convert inputs to lists of strings
+        if parameters is not None:
+            parameters = {prm.name if isinstance(prm, Parameter) else prm for prm in parameters}
```

Comment:
Input type annotation says only str can be accepted.

Reply:
Fixed here: 113150a